content/e20co/drone-marathon/index.md

```diff
@@ -78,7 +78,19 @@ Will result in:
 
 # Secrets
 
-Not Implemented.
+The Marathon plugin supports reading credentials from the Drone secret store. This is highly recommended instead of storing credentials in the pipeline configuration in plain text.
+
+```diff
+pipeline:
+  marathon:
+    image: e20co/drone-marathon:0.5
+    server: http://marathon.mesos:8080
+    values:
+-     DATABASE_PASSWORD: SomeSecretPassword
++     DATABASE_PASSWORD: ${DATABASE_PASSWORD}
```

Comment:
string interpolation of secrets using `${var}` doesn't work in 0.5 (although there is discussion about adding it back). Secrets are injected directly into the container as environment variables, similar to `docker run -e DATABASE_PASSWORD=foo`